core/src/main/java/io/camunda/migrator/RuntimeMigrator.java

```diff
@@ -86,6 +95,14 @@ public void migrate() {
     activateMigratorJobs();
   }
 
+  protected List<String> listSkippedProcessInstances() {
+   return new Pagination<String>()
+        .batchSize(batchSize)
+        .maxCount(idKeyMapper::findSkippedProcessInstanceIdsCount)
+        .page(offset -> idKeyMapper.findSkippedProcessInstanceIds(0, batchSize))
+        .toList();
```

Comment:
/q When passing a `#callback` function that delegates to the printer, we can avoid loading all IDs into the memory at once. Printing in batches would avoid this.